tt_metal/impl/dispatch/command_queue.cpp

```diff
@@ -337,7 +339,8 @@ EnqueueProgramCommand::EnqueueProgramCommand(
     this->unicast_cores_launch_message_wptr = unicast_cores_launch_message_wptr;
 }
 
-void EnqueueProgramCommand::assemble_preamble_commands(ProgramCommandSequence& program_command_sequence, std::vector<ConfigBufferEntry>& kernel_config_addrs) {
+void EnqueueProgramCommand::assemble_preamble_commands(
+    ProgramCommandSequence& program_command_sequence, const std::span<ConfigBufferEntry> kernel_config_addrs) {
```

Comment:
Should use tt::stl::Span instead of std::span for all usages in metal

Reply:
Done.